Sources/Datadog/FeaturesIntegration/WebView/WebEventBridge.swift

```diff
@@ -51,22 +51,13 @@ internal class WebEventBridge {
             throw WebEventError.missingKey(key: Constants.eventKey)
         }
 
-        if eventType == Constants.eventTypeLog ||
-            eventType == Constants.eventTypeInternalLog {
-            if let consumer = logEventConsumer {
-                try consumer.consume(
-                    event: wrappedEvent,
-                    internalLog: (eventType == Constants.eventTypeInternalLog)
-                )
-            } else {
-                DD.logger.warn("A WebView log is lost because Logging is disabled in the SDK")
-            }
+        if eventType == Constants.eventTypeLog || eventType == Constants.eventTypeInternalLog {
+            try logEventConsumer.consume(
+                event: wrappedEvent,
+                internalLog: (eventType == Constants.eventTypeInternalLog)
```

Comment:
This makes me realise that `internalLog` is a leftover after https://github.com/DataDog/dd-sdk-ios/pull/819 and it leaves a dead path in the code, no? There is no use of `internalLog` bool inside this `.consume(event:internalLog:)` method as far as I can see. How about cleaning it up 🏕️?

Reply:
Ah yes, I noticed it but forgot to mention it. Browser SDK also migrated to RUM Telemetry, so we might no longer receive `internal_log` event type. I will double check with them and remove this 👍